webapp/utils/constants.jsx

```diff
@@ -849,7 +849,8 @@ export const Constants = {
     MENTION_MEMBERS: 'mention.members',
     MENTION_NONMEMBERS: 'mention.nonmembers',
     MENTION_SPECIAL: 'mention.special',
-    DEFAULT_NOTIFICATION_DURATION: 5000
+    DEFAULT_NOTIFICATION_DURATION: 5000,
+    STATUS_INTERVAL: 15000
```

Comment:
0/5 I can change it to be higher for now, but we need to do loadtests that represent more realistic loads to see how low this can be before it starts causing issues